device/inspur/x86_64-inspur_sc5630el-r0/minigraph.xml

```diff
@@ -0,0 +1,1194 @@
+<DeviceMiniGraph xmlns:i="http://www.w3.org/2001/XMLSchema-instance" xmlns="Microsoft.Search.Autopilot.Evolution">
```

Comment:
please remove minigraph, it is not needed.

Reply:
OK, done